### PR TITLE
HardwareTimer: getTimerClkFreq() add comment to avoid confusion

### PR DESCRIPTION
**Pull request by ABOSTM**
**Summary**

HardwareTimer: getTimerClkFreq() add comment to avoid confusion

### CODE REVIEW

File: cores/arduino/HardwareTimer.cpp [hunk 1]
```diff
@@ -1300,6 +1300,7 @@ uint32_t HardwareTimer::getTimerClkFreq()
       /* case RCC_APB2_DIV1: */
       case RCC_APB2_DIV2:
       case RCC_APB2_DIV4:
+        /* Note: it such cases, HCLK = (APBCLK * DIVx) */
```
Comment (fpistm) [inline cores/arduino/HardwareTimer.cpp:1303]:
```suggestion
        /* Note: in such cases, HCLK = (APBCLK * DIVx) */
```

File: cores/arduino/HardwareTimer.cpp [hunk 2]
```diff
@@ -1316,7 +1317,7 @@ uint32_t HardwareTimer::getTimerClkFreq()
       case RCC_APB1_DIV2:
       /* case RCC_APB2_DIV1: */
       case RCC_APB2_DIV2:
-        // uwTimclock*=1;
+        /* Note: it such cases, HCLK = (APBCLK * DIVx) */
```
Comment (fpistm) [inline cores/arduino/HardwareTimer.cpp:1320]: dito

File: cores/arduino/HardwareTimer.cpp [hunk 3]
```diff
@@ -1349,6 +1350,7 @@ uint32_t HardwareTimer::getTimerClkFreq()
       case RCC_HCLK_DIV1:
       case RCC_HCLK_DIV2:
       case RCC_HCLK_DIV4:
+        /* Note: it such cases, HCLK = (APBCLK * DIVx) */
```
Comment (fpistm) [inline cores/arduino/HardwareTimer.cpp:1353]: dito

File: cores/arduino/HardwareTimer.cpp (hunk 1 above)
Comment (ABOSTM) [reply cores/arduino/HardwareTimer.cpp]: Done

**Pull request merged**